scripts/uploadWhitelist.js

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable no-console */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+require("babel-register");
+const d3 = require("d3-dsv");
+const fs = require("fs");
+const confirm = require("node-ask").confirm;
+const path = require("path");
+
+const Commitment = artifacts.require("Commitment");
+
+const tokenEnum = {
+  EUR: 2,
+  ETH: 1
+};
+
+const Q18 = new web3.BigNumber(10).pow(18);
+
+const isAddress = address => {
+  if (!web3.isAddress(address))
+    throw new Error(
+      `Investor with Address:${address} has wrong address format!!`
+    );
+  return address;
+};
+const isCurrency = (currency, address) => {
+  if (currency !== "EUR" && currency !== "ETH")
+    throw new Error(
+      `Investor:${address} and currency:${currency} has wrong currency format!!`
+    );
+  return tokenEnum[currency];
+};
+const getAmount = (amount, address) => {
+  const investAmount = Number.parseFloat(amount.replace(/^\D+/g, ""));
```

Comment:
we need to use better parser. this one accepts any messy string starting with a number